drivers/network/tcpip/ip/transport/tcp/tcp.c

```diff
@@ -410,7 +410,14 @@ NTSTATUS TCPConnect
     Status = TCPTranslateError(LibTCPConnect(Connection,
                                                 &connaddr,
                                                 RemotePort));
-
+    if (!NT_SUCCESS(Status))
+    {
+        LockObject(Connection);
+        RemoveEntryList(&Bucket->Entry);
+        UnlockObject(Connection);
+        ExFreeToNPagedLookasideList(&TdiBucketLookasideList, Bucket);
+    }
+    
```

Comment:
```suggestion
    }

```

Reply:
Hum, please restore the empty line.

Reply:
Serge is talking about the empty line **under** the closing curly brace.